CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Add additional options to the build command for testing different fakeroot
   modes: `--userns` like the action flag and hidden options `--ignore-subuid`,
   `--ignore-fakeroot-command`, and `--ignore-userns`.
+- Add additional profile for testing unprivileged overlayfs driver.
```

Comment:
There's no need for a CHANGELOG entry here since only e2e tests are affected.

Reply:
got it!